lib/actions/device.coffee

```diff
@@ -278,20 +294,62 @@ exports.init =
 					return callback(new Error("Invalid application: #{options.application}"))
 
 				return callback(null, params.device) if params.device?
-				visuals.patterns.selectDrive(callback)
+				drivelist.list (error, drives) ->
+					return callback(error) if error?
+
+					async.reject drives, drivelist.isSystem, (removableDrives) ->
+
+						if _.isEmpty(removableDrives)
+							return callback(new Error('No available drives'))
+
+						form.ask
+							message: 'Drive'
+							name: 'drive'
+							type: 'list'
+							default: _.map removableDrives, (item) ->
```

Comment:
This means that the default selection for a drive is a list of drives, which makes no sense. This property should be `choices` instead.